.travis.yml

```diff
@@ -37,4 +37,3 @@ script:
         echo "Building all recipes.";
         source ./.CI/build_all;
       fi
-
```

Comment:
Please merge PR ( https://github.com/conda-forge/staged-recipes/pull/1615 ) if this is needed. Otherwise please close it.